.buildkite/scripts/snyk/plugins-scan/generate-steps.py

```diff
@@ -132,19 +132,29 @@ def parse_plugin_entry(entry) -> list:
 def generate_snyk_step(plugin_name: str, branch: str) -> dict:
     """Generate a Buildkite step for running snyk monitor on a plugin."""
     step_key = slugify_bk_key(f"snyk-{plugin_name}-{branch}")
+    if plugin_name == 'logstash-filter-elastic_integration':
+        repo_url = f"https://github.com/elastic/{plugin_name}.git"
+    else:
+        repo_url = f"https://github.com/logstash-plugins/{plugin_name}.git"
 
     command = f"""#!/bin/bash
 set -euo pipefail
 
-echo "--- Downloading snyk..."
-curl -sL --retry-max-time 60 --retry 3 --retry-delay 5 https://static.snyk.io/cli/latest/snyk-linux -o snyk
-chmod +x ./snyk
+export JAVA_HOME="/opt/buildkite-agent/.java/adoptiumjdk_21"
+export PATH="/opt/buildkite-agent/.java/bin:$JAVA_HOME:$PATH"
```

Comment:
Can we just source https://github.com/elastic/logstash/blob/main/.buildkite/scripts/common/vm-agent-multi-jdk.sh like we do elsewhere for prepping java env?